CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
   - pxe_stack: fix issues with hostname not set during kickstart on RHEL 8.3 (#522)
   - set_hostname: add fqdn capability (#543)
   - time: allow to set sysconfig OPTIONS for chronyd (#552)
+  - time: allow to additional networks for server to reply (#555)
```

Comment:
There should be a typo here: `allow to _verb_ additional` (configure, define, add, your choice :smiley:)